JS/Lesson30/savas/my-app/src/components/Ex9/index.jsx

```diff
@@ -0,0 +1,40 @@
+// ## Exercise 9: Multiple Buttons
+// Create a component with multiple buttons. Change the background color of the component based on which button is clicked.
+
+export const Ex9 = () => {
+
+    function handleClickYellow () {
+        document.getElementById('root').style.backgroundColor = "yellow";
```

Comment:
When using React, we normally do not select elements as we did in pure JS (`document.getElementById('root')`).
Since the background that you want to change is not accessible from here, you'd normally use state management on the application level here (Redux or ContextApi). 
No need to change anything in this exercise right now but avoid using pure JS element selector going forward. :)

Reply:
Ok, I understood :)